### PR TITLE
v2.0.0 Release Docs Updates

### PR DESCRIPTION
**Pull request by faberga**

### CODE REVIEW

**Comment by faberga** [comment]
Jenkins run tests

File: CHANGES.md [hunk 1]
```diff
@@ -4,9 +4,10 @@ HElib 2.0.0, January 2021
 
 December-January 2021
 ---------------------
-* Changes to Context API
-* Must use ContextBuilder to build context
-* New serialization APIs: binary and JSON
+* v2.0.0 contains changes of APIs that make it not backwards compatible with v1.x.x
```
Comment (hamishun) [inline CHANGES.md:7]:
```suggestion
* Contains API changes that are not backwards compatible with v1.x.x
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ tests, designing and coding new examples or writing tutorials are all examples
 of helpful contributions. 
 
 A contribution to HElib can be initiated through GitHub pull request (PR).
-HElib is written in C++14 and uses `clang-format` for the formatting of the code.
+HElib is written in C++17 and uses `clang-format` for the formatting of the code.
 Requiremens and installation instructions can be found in [INSTALL.md](INSTALL.md).
```
Comment (hamishun) [inline CONTRIBUTING.md:9]:
```suggestion
Requirements and installation instructions can be found in [INSTALL.md](INSTALL.md).
```

File: examples/README.md [hunk 3]
```diff
@@ -2,17 +2,20 @@
...
-The `examples` directory provides standalone example programs expressing the
-various APIs as well as a simple use-case using HElib. The example programs
-provided currently only use the BGV scheme.
+The `examples` directory contains tutorials and standalone example programs
+expressing the various APIs as well as a simple use-case using HElib. 
 
 ## What is provided
 
-Currently the examples provided comprise:
+The [tutorials](tutorial/) cover primarely the CKKS scheme and comprise 8 documented axamples
```
Comment (hamishun) [inline examples/README.md:10]:
```suggestion
The [tutorials](tutorial/) cover primarily the CKKS scheme and comprise 8 documented examples 
```

**Review by faberga — COMMENTED** [review]
Good catch.

**Review by jlhcrawford — APPROVED** [review]

**Review by hamishun — APPROVED** [review]

**Comment by hamishun** [comment]
@faberga Wait for Jenkins

**Comment by hamishun** [comment]
Jenkins run tests

**Comment by faberga** [comment]
> @faberga Wait for Jenkins

I was checking the server and saw it being triggered.

**Comment by faberga** [comment]
Jenkins tests passed.

**Pull request merged**